test/unit/org/apache/cassandra/cql3/conditions/ColumnConditionTest.java

```diff
@@ -136,6 +136,22 @@ private static boolean conditionApplies(List<ByteBuffer> rowValue, Operator op,
         return bound.appliesTo(newRow(definition, rowValue));
     }
 
+    private static boolean conditionContainsApplies(List<ByteBuffer> rowValue, Operator op, ByteBuffer conditionValue)
+    {
+        ColumnMetadata definition = ColumnMetadata.regularColumn("ks", "cf", "c", ListType.getInstance(Int32Type.instance, true));
```

Comment:
Note to myself: the guy that created this test class did not do a really good job. He forgot to test for non-frozen collections.

Reply:
@Ankou76ers no worries, I am not asking you to fix my mistakes. Your changes are fine :-)